lib/rubocop/path_util.rb

```diff
@@ -1,17 +1,29 @@
 # frozen_string_literal: true
 
+require 'rbconfig'
+
 module RuboCop
   # Common methods and behaviors for dealing with paths.
   module PathUtil
     module_function
 
+    def windows?
+      (RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/)
```

Comment:
Bundler detects Windows a bit differently. I'm not sure which is "right".

```ruby
RbConfig::CONFIG["host_os"] =~ /(msdos|mswin|djgpp|mingw)/
```

https://github.com/bundler/bundler/blob/b3cb9516aaf83cf865c9ed76c52ec4d0d182015f/lib/bundler/constants.rb#L3